### DISCUSSION TITLE
Psychosis/Watchers gaze makes you walk towards the airlock at 100%

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5980] [Ideas and suggestions]
Essentially, unless you acted against it, psychosis would make your character use the AI nodes to navigate towards an airlock and swim out, making it far more dangerous. Your character would follow your inputs normally, however if you stopped trying to walk or run somewhere, you would begin moving towards the airlock.

**Comment by ViktorShield** [comment]
Watcher is not Mezmer from Subnautica

**Comment by Layarion** [comment]
I think as is, Phychosis is a just a joke. It's too easy to tell the fake from the real. So to the end of making it not a joke, I agree with this; but I think some other things could be done as well.

1) add in fake health loss with those fake fires. when they stand in the fake fire, they take damage that lowers the healthbar they see. if it hits zero, they get stunned. 
2) the fake broken items, when a player tries to repair them, it should let them repair them and make the player think they actually fixed the devices. in reality they didn't repair the device and other players can see their character just standing in front of a non-broken device doing repair animations for no reason.
3) it should make them see enemies, and some of those enemies should be friendly crew-mates.

basically, the physcosis is a joke because it's too easy to ignore or tell if thing is real or not.

**Reply by MateusJK15** [reply]
P E R F E C T

**Reply by Likvidus** [reply]
I like these ideas. You can also add the effect of high pressure and visible damage of the hull during false flooding (with stunning the character instead of barotrauma at the end). At the same time, other characters in this "flooded" compartment become invisible for maximum effect. At a high level of psychosis, "images" of raptors or crawlers appear, which attack the character and stun him ( if they are damaged, they disappear without a trace). As an addition, you can add the opposite effect. For example: there is a fire or flooding, but a person will not see it because of psychosis.
